### PR TITLE
adding link to original viewer; code cleanup

### PR DESCRIPTION
**Pull request by sravotto**
Hi Radu, I added a link to the original viewer in the toolbar, so it's easy to switch. 
Pointed the example to the live page.
Reformatted some code as well...

### CODE REVIEW

File: altdistsqlplan/css/style.css [hunk 1]
```diff
@@ -95,7 +90,4 @@ body {
 .details th > div > span {
     border-bottom: 1px solid #ccc;
     padding: 5px 10px;
-}
-
-
-
+}
```
Comment (RaduBerinde) [inline altdistsqlplan/css/style.css]: [nit] missing end of line character

File: altdistsqlplan/decode.html [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
 
+<head>
+  <script>location.replace('index.html' + window.location.hash);</script>
 </head>
+
 <body>
-  
 </body>
+
+</html>
```
Comment (RaduBerinde) [inline altdistsqlplan/decode.html:11]: ditto

File: altdistsqlplan/example.html [hunk 3]
```diff
@@ -1 +1 @@
...
...UiF43ItwXSosswo0IB__-9WcUTSGIomnSPoyiqetG0TRw508rP8PCjLGDexMdog51OsTc_eFXpNMRhIgxMlLFRiwyDZlORTazuf_8PTNJcaeQBJ6EiPqO77eK7f69hvm-91MLnaSH2MEjPkUjGAl5j_hwKGKuwdibLvqQ2_tcxzegkJjosQmKBtjB-Q3_7xR3XD46uNiaIao0HwAOyaNTH_U3g4GEAddCNtgy6B1T1J5MQJr48tWb7qerbu_8qntxfHzQIa9MKS5ODjrUPL3tXXTPZ88whXiyABXNQSxnRlYSK-X0FGb_Ht1wdbMS4eXjU950bd5PfkSRT9nPj4UjCzit54FzdnFydWTgYWZ1ClkC0pxzUIc2OqwCMi8o2msjaGxtI7AagEyyKkgq0eiK12LcaC8DsVLp6qi9ctTttVF7S1GT-qJBaonGvjTDiM-Yy1TlfowPZiz8DxVFU87gl82-SBRN-7RYYsSzBBEk9A3I9Xrk-952etRqO57nreoRoTvKUWtFjmZX1JOjDfVcYJz3cnLk1ZMjf49yRPYrR_7XlqO2W01sWgatWVOOaH1i07rE3icfDb8Vz9C15FmcqljM_QbQMr_XUdR_H0VTct2p59m97tgnjfb2k0bgVk0a_l4njcCtSe0NFV3o3uY3nzTKf5rPoTbdL7WDr01tWnfSIDtMGhW5noIai0xBaeKo9uwatCAZQIGuEhMZw0cp4vyaYtnLz-UbCShdWEmxOMoKkwlw8TApHyaLh-nSYZJHoyvnHcZY3nHVXCNzqhmz-jJEGkZjlKTqM5ooPjDURlVMdMvQ5xzcJoPW2gwy0HdCfkZDriGL70NUfOHMt-94qku5JaBApnyY_s4X24YVbTM_OJOmGNJbkzpdMM3laW5jTZqnPbePQBk0ll6p_b2wDSrtF0elaQXFtYDiboKE7tgo1MoTtgxJ-TCzk8y1s8yznm7aDzd3pGj9UnqetcFday2ZrcEDby8NzlaraUdlPe1XZmCPWoSL7Shcz89gHxS115UE1roSW13pRpbWqitZRaW1DUt3QsX3fQsqvl3OmxZUWl6wCRWv9qfDMiq-FZXALiDBiwsI8wrQ1kD6PfaZHZQtxga6DpRW0wYKJbuPDaz1UmODHZX1mlRW1Vly1aJKX3AabH9j-aDezs1O_f1Mg_k32uMP_wUAAP__K-8jbQ=="/>
```
Comment (RaduBerinde) [inline altdistsqlplan/example.html]: Why not just `url=index.html#` ?

**Review by RaduBerinde — APPROVED** [review]

File: altdistsqlplan/example.html (hunk 3 above)
Comment (sravotto) [reply altdistsqlplan/example.html]: I'll fix that.

**Pull request merged**